package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "scripts": {
     "build": "NODE_ENV=production ./node_modules/.bin/webpack --config ./config/webpack.config.js",
-    "postinstall": "npm run build",
+    "postinstall": "echo 'npm run build'",
```

Comment:
Should we remove this entirely if it's not needed?

Reply:
@lbraun If it's not needed as part of the deploy process, I would vote yes.

Reply:
@lbraun I reverted this to how it was previously in case it does break deploy.  If it's not needed for that process, I recommend that we remove it entirely.

Reply:
Sounds good, thanks, will have to test that out.